### PR TITLE
feat: supported to specify slave zones.

### PR DESCRIPTION
**Pull request by zhaiyongsheng0123**

### CODE REVIEW

File: alicloud/resource_alicloud_db_instance.go [hunk 1]
```diff
@@ -752,6 +770,14 @@ func buildDBCreateRequest(d *schema.ResourceData, meta interface{}) (*rds.Create
 		request.SecurityIPList = strings.Join(expandStringList(d.Get("security_ips").(*schema.Set).List())[:], COMMA_SEPARATED)
 	}
 
+	slaveZoneIds := d.Get("slave_zone_ids").([]interface{})
```
Comment (zql-tqy) [inline alicloud/resource_alicloud_db_instance.go:773]: 选填参数使用 d.getok() 进行获取

File: website/docs/d/db_instances.html.markdown [hunk 2]
```diff
@@ -77,3 +77,5 @@ The following attributes are exported in addition to the arguments listed above:
   * `connection_string` - (Available in 1.70.3+) RDS database connection string.
   * `instance_storage` - (Available in 1.70.3+) User-defined DB instance storage space.
   * `db_instance_storage_type` - (Available in 1.70.3+) The storage type of the instance.
+  * `master_zone` - (Available in 1.88.1+) The master zone of the instance.
```
Comment (zql-tqy) [inline website/docs/d/db_instances.html.markdown:80]: 1.89.0

File: website/docs/r/db_instance.html.markdown [hunk 3]
```diff
@@ -140,6 +140,7 @@ The multiple zone ID can be retrieved by setting `multi` to "true" in the data s
 * `auto_upgrade_minor_version` - (Optional, Available in 1.62.1+) The upgrade method to use. Valid values:
    - Auto: Instances are automatically upgraded to a higher minor version.
    - Manual: Instances are forcibly upgraded to a higher minor version when the current version is unpublished.
+* `slave_zone_ids` - (ForceNew) List of slave zone ids can specify slave zone ids when creating the high-availability or enterprise edition instances. Meanwhile, `vswitch_id` needs to pass in the corresponding vswitch id to the slave zone by order. For example, `zone_id = "zone-a"` and `slave_zone_ids = ["zone-c", "zone-b"]`, then the `vswitch_id` must be `"vsw-zone-a,vsw-zone-c,vsw-zone-b"`. The list contains up to 2 slave zone ids, separated by commas. From version 1.88.1, it supports to specify slave zones.
```
Comment (zql-tqy) [inline website/docs/r/db_instance.html.markdown:143]: （Optional, ForceNew, Available in 1.89.0+）

Comment (zql-tqy) [inline website/docs/r/db_instance.html.markdown:143]: 增加的出参 master_zone 没有添加文档

**Comment by xiaozhu36** [comment]
HI @zhaiyongsheng0123 Thanks for your contribution. Considering internal spec, we have supported this feature by PR #2924 and it has been published in the version 1.101.0. Please check it.

**Pull request closed without merging**